bin/liquidator/src/rebalance.rs

```diff
@@ -298,6 +307,7 @@ impl Rebalancer {
     }
 
     async fn rebalance_tokens(&self) -> anyhow::Result<()> {
+        self.settle_and_close_all_openbook_orders().await?;
```

Comment:
Might doing a cancel+settle of all order on each end up too frequent? I guess due to the sleep workaround in the rebalance_job it's at most once every 10s.

Reply:
It will not do anything if there is no open orders account or if there is an active order for the token